apps/web-site/src/app/App.vue

```diff
@@ -172,6 +172,9 @@ body {
 p {
   margin: 0;
 }
+.pm-p {
```

Comment:
Since I changed the prosemirror wrapper element to div, we need this to make line breaks work. There's probably a better way!